extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -59,4 +59,43 @@ public function approveDraft( Title $title ) {
 		// Save to parent page
 		$page->doEdit( $draftContent, wfMessage( 'templatedraft-approval-summary' )->inContentLanguage()->plain() );
 	}
+
+	public function markTemplateAsNotInfobox() {
+		/**
+		 * First, validate a request
+		 */
+		if ( !$this->wg->User->isPowerUser()
```

Comment:
We should tie this to a user right validated via `User::userCan()` (quoting method from memory, please verify) and add it to the following groups:
* powerusers
* staff
* helpers
* vstf
* voldev